<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Praxis</class>
 <widget class="QWidget" name="Praxis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>110</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>524</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>524</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>50</height>
    </rect>
   </property>
   <widget class="MyTextBrower" name="_pQuestionLab">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblAnswer">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="MyTextBrower" name="_pQuestionBtn_1">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>62</y>
     <width>275</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>275</width>
     <height>18</height>
    </size>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="MyTextBrower" name="_pQuestionBtn_2">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>90</y>
     <width>275</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>275</width>
     <height>18</height>
    </size>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="MyTextBrower" name="_pQuestionBtn_3">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>62</y>
     <width>275</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>275</width>
     <height>18</height>
    </size>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="MyTextBrower" name="_pQuestionBtn_4">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>90</y>
     <width>275</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>275</width>
     <height>18</height>
    </size>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTextBrower</class>
   <extends>QTextBrowser</extends>
   <header>mytextbrower.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
